contribs/drt/src/main/java/org/matsim/contrib/drt/analysis/DrtAnalysisControlerListener.java

```diff
@@ -120,7 +120,8 @@ public void notifyIterationEnds(IterationEndsEvent event) {
 
 		double rejectionRate = (double)drtRequestAnalyzer.getRejectedRequestSequences().size()
 				/ drtRequestAnalyzer.getRequestSubmissions().size();
-		String tripsSummarize = DrtTripsAnalyser.summarizeTrips(trips, drtVehicleStats.getTravelDistances(), ";");
+		String tripsSummarize = DrtTripsAnalyser.summarizeTrips(trips, drtVehicleStats.getTravelDistances(),
+				";", drtRequestAnalyzer.getSumFaresNotReferencingATrip());
```

Comment:
Yes, that's right. I renamed "trip" to "leg" in all drt analysis classes. I did not for the drt fare classes, because this brakes all existing configs with the field minFarePer_Trip_. Should I rename that in this PR, too?

Reply:
This will be a braking change, but perhaps of a minor impact, so I would say yes.

Reply:
Ok, so I will rename the drt fares, but probably in a separate PR